Controller/SectionController.php

```diff
@@ -46,11 +46,13 @@ public function listAction()
     {
         $sectionList = $this->sectionService->loadSections();
         $contentCountBySectionId = [];
+        $canDeleteBySectionId = [];
```

Comment:
maybe i'd try another variable naming here ```can``` makes me expect a boolean here, but it's an array.
i'm +1 anyway.